samples/sitecore-embedded-jss-app/src/app/components/Wizard.js

```diff
@@ -14,6 +14,10 @@ class Wizard extends React.Component {
     this.onFormValueChange = this.onFormValueChange.bind(this);
   }
 
+  componentDidMount() {
+    console.log('Here should be wizard');
+  }
+
```

Comment:
Remove this componentDidMount/console.log?

Reply:
@ambrauer  It's expected, we had it in StepZillaPatched

Reply:
nvm, it's not expected, will remove